salt/runners/vistara.py

```diff
@@ -74,7 +74,7 @@ def _get_vistara_configuration():
                  "Exception enountered: {0}.format(exc)"
             )
             return False
-    log.error(
```

Comment:
this module shouldn't even be logging on a function called from `__virtual__`...

It should return `(False, error message)`...